mobile-app/app/screens/AppNavigator/screens/Portfolio/components/TokenBreakdownDetailsV2.tsx

```diff
@@ -0,0 +1,436 @@
+import { useTokenPrice } from '../hooks/TokenPrice'
+import { PortfolioButtonGroupTabKey } from './TotalPortfolio'
+
+interface TokenBreakdownDetailProps {
+  hasFetchedToken: boolean
+  lockedAmount: BigNumber
+  lockedValue: BigNumber
+  availableAmount: BigNumber
+  availableValue: BigNumber
+  dfiUtxo?: WalletToken
+  dfiToken?: WalletToken
+  testID: string
+  denominationCurrency: string
+  token: WalletToken
+  usdAmount: BigNumber
+  pair?: PoolPairData
+}
+
+export function TokenBreakdownDetailsV2 (props: TokenBreakdownDetailProps): JSX.Element {
+  const displayDenominationCurrency = props.denominationCurrency === PortfolioButtonGroupTabKey.USDT ? 'USD' : `${props.denominationCurrency}`
+  const lockedToken = useTokenLockedBalance({ displaySymbol: props.token.displaySymbol, denominationCurrency: props.denominationCurrency }) as LockedBalance ?? { amount: new BigNumber(0), tokenValue: new BigNumber(0) }
+  const loanTokens = useSelector((state: RootState) => state.loans.loanTokens)
+  const collateralTokens = useSelector((state: RootState) => state.loans.collateralTokens)
+  const hasLockedBalance = useMemo((): boolean => {
+    return collateralTokens.some(collateralToken => collateralToken.token.displaySymbol === props.token.displaySymbol) ||
+      loanTokens.some(loanToken => loanToken.token.displaySymbol === props.token.displaySymbol)
+  }, [props.token])
+  const { getTokenPrice } = useTokenPrice()
+
+  // LP token calculations
+  const { poolpairs: pairs } = useSelector((state: RootState) => state.wallet)
+  const poolPairData = pairs.find(
+    (pr) => pr.data.symbol === (props.token as AddressToken).symbol
+  )
+  const mappedPair = poolPairData?.data
+  const toRemove = new BigNumber(1)
+    .times((props.token).amount)
+    .decimalPlaces(8, BigNumber.ROUND_DOWN)
+  const ratioToTotal = toRemove.div(mappedPair?.totalLiquidity?.token ?? 1)
+  const tokenATotal = ratioToTotal
+    .times(mappedPair?.tokenA.reserve ?? 0)
+    .decimalPlaces(8, BigNumber.ROUND_DOWN)
+  const tokenBTotal = ratioToTotal
+    .times(mappedPair?.tokenB.reserve ?? 0)
+    .decimalPlaces(8, BigNumber.ROUND_DOWN)
+  const getUSDValue = (
+    amount: BigNumber,
+    symbol: string,
+    isLPs: boolean = false
+  ): BigNumber => {
+    return getTokenPrice(symbol, amount, isLPs)
+  }
+
+  return (
+    <ThemedViewV2
+      light={tailwind('border-mono-light-v2-300')}
+      dark={tailwind('border-mono-dark-v2-300')}
+    >
+      {/* To display options for locked in collateral/vault and available amount */}
+      {
+        hasLockedBalance && !lockedToken.amount.isZero() && (
+          <View style={tailwind('pb-4')}>
+            <TokenBreakdownDetailsRow
+              testID={`${props.token.symbol}_locked`}
+              amount={lockedToken.amount.toFixed(8)}
+              label='Locked in vault(s)'
+              hasFetchedToken={props.hasFetchedToken}
+              labelTextStyle={tailwind('font-normal-v2')}
```

Comment:
Can we set this as default style? It seems all TokenBreakdownDetailsRow uses this